INSTALL

```diff
@@ -48,7 +48,7 @@ WHAT DO YOU NEED ?
    Opam (https://opam.ocaml.org/) is recommended to install ocaml and
    the corresponding packages.
 
-   $ opam install ocamlfind camlp5 lablgtk-extras
+   $ opam install ocamlfind camlp5 lablgtk-extras ounit
```

Comment:
Don't add this. You may create a new category "dependency of the test-suite" if you want. There are many more "optional" dependencies in Coq: for the documentation, `python` for the timing targets, `rsync` and `which` for the test-suite.

Reply:
OK.